src/menu/DatasetCard.tsx

```diff
@@ -45,14 +49,15 @@ export default function DatasetCard({name, headerIcon, tabs, viewId, dataSource}
 
   const loadNamedSets = React.useMemo<() => Promise<INamedSet[]>>(() => {
     return () => RestStorageUtils.listNamedSets(dataSource.idType);
-  }, [dataSource.idType]);
+  }, [dataSource.idType, dirtyNamedSets]);
 
   const predefinedNamedSets = useAsync(loadPredefinedSet);
   const me = UserSession.getInstance().currentUserNameOrAnonymous();
   const namedSets = useAsync(loadNamedSets);
   const myNamedSets = {...namedSets, ...{value: namedSets.value?.filter((d) => d.type === ENamedSetType.NAMEDSET && d.creator === me)}};
   const publicNamedSets = {...namedSets, ...{value: namedSets.value?.filter((d) => d.type === ENamedSetType.NAMEDSET && d.creator !== me)}};
   const filterValue = (value: INamedSet[], tab: string) => value?.filter((entry) => entry.subTypeValue === tab);
```

Comment:
It looks like the `onOpen` function from https://github.com/Caleydo/tdp_publicdb/pull/134/files#diff-ba4f4a853dd3cfba82ee75d7d7bd4d28529fb4ebb11a8a266186f853c38151dfR58-R67 got lost. Please check again and re-add. See also https://github.com/Caleydo/ordino/pull/273#pullrequestreview-617912314.